www/src/providers/Auth0ProviderWithHistory.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useHistory } from 'react-router-dom';
+import { AppState, Auth0Provider } from '@auth0/auth0-react';
+
+const Auth0ProviderWithHistory: React.FC = ({ children }) => {
+    const domain = process.env.REACT_APP_AUTH0_DOMAIN ?? '';
+    const clientId = process.env.REACT_APP_AUTH0_CLIENT_ID ?? '';
```

Comment:
In the future we should probably move this to a config module like we do in the `api`. Let us just do stuff like `config.auth0Domain`.

Reply:
Sounds good, let me merge that in here actually